stable/mlrun/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mlrun
-version: 0.5.0
-appVersion: 0.5.0
+version: 0.5.1
+appVersion: 0.5.2
```

```suggestion
appVersion: ">=0.5.2"
```
Helm charts change much less often than app releases, I'm guessing this is meant to support 0.5.3 as well and onwards, right?

---

How can I know whether this chart supports a version that is not released yet ?
Also, ~~I've never seen~~ just checked, there is no chart doing such a thing for the `appVersion`, it is advised doing it for the `kubeVersion`:
> A SemVer range of compatible Kubernetes versions (optional)
but not for the `appVersion`, the docs says:
> The version of the app that this contains (optional). This needn't be SemVer.

---

> "How can I know whether this chart supports a version that is not released yet ?"

The answer to this is "intent"

> "Also, I've never seen just checked, there is no chart doing such a thing for the "

Check again. In this repo :) It is the internal convention here on many charts.

> "This needn't be SemVer."

"needn't" means doesn't have to be. Not that it can't :)


tl;dr - do you not intend (or can't) for this chart to serve future app versions ?

---

@Hedingber  - IIUC, You do not want to guarantee compatibility here, so this is why you only state the pinpoint version.
I'm fine with that but be aware that those charts (like you mentioned offline) are designed to serve more than one released version. So keep this in mind, this will probably serve more than just 0.5.2 and while it's not functionally enforced today, it will one day and we will use this chart "out of it's intended parameters"